code/package.json

```diff
@@ -12,7 +13,10 @@
     "eslint-plugin-react-hooks": "^2.3.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "^3.3.0"
+    "react-redux": "^7.2.2",
+    "react-scripts": "^3.3.0",
+    "save": "^2.4.0",
```

Comment:
We were  wondering about this dependency. Were you trying to save the data locally?

Reply:
Actually, we don't remember installing this or where it comes from :)